src/Angor/Client/Pages/Create.razor

```diff
@@ -1214,7 +1214,9 @@
 
             if (!response.Success)
             {
-                notificationComponent.ShowErrorMessage(response.Message);
+                var transactionException = new Exception(response.Message);
```

Comment:
why do all this extra work that is similar everywhere.
Just allow to pass a message in to the notificationComponent without an exception and also log the error inside the component.

Reply:
![image](https://github.com/user-attachments/assets/cbd47e51-cc68-44a8-9054-cf9229ea6fad)
To avoid displaying such errors

Reply:
you can pass an error message and and exception, if no error message just write "Error Happened" and allow the user to expand